src/handshake.ts

```diff
@@ -0,0 +1,19 @@
+import { bytes, bytes32 } from "./types/basic";
+import { NoiseSession, XXHandshake } from "./xx";
+import { KeyPair } from "./types/libp2p";
+
+export class Handshake {
+  static async runXX(
```

Comment:
I would convert this to constructor and have methods like init/exchange/finish like in secio

Reply:
later on it would allow us to combine multiple handshakes